faq/windows.po

```diff
@@ -420,14 +420,18 @@ msgstr ""
 
 #: ../Doc/faq/windows.rst:278
 msgid "How do I check for a keypress without blocking?"
-msgstr ""
+msgstr "Comment puis-je vérifier qu'une touche a été pressée sans bloquer ?"
```

Comment:
Ca serait pas plutot `sans bloquer la suite de l'execution de mon programme` (ou quelque chose semblable) ? Pour avoir plus de sens dans cette phrase

Reply:
On peut mais c'est ultra verbeux et la doc d'origine elle dit bien juste "bloquer". Je voudrais bien un avis tiers pour trancher. @JulienPalard , @christopheNan ?

Reply:
Je suis d'accord, la phrase d'origine serait plus interessante a changer

Reply:
Je pense que la version en anglais est bonne, du moins d'un point de vue de développeur. On retrouve ce "without blocking" un peu partout, un `man -K 'without blocking'` en sort certaines des pages de man comme :

> The file descriptor is writable (the select(2) writefds argument; the poll(2) POLLOUT flag) if it is possible to write a value
of at least "1" without blocking.

(eventfd_read)

> You can also get shared locks or attempt to get a lock without blocking.

(IPC::Shareable)

Ce que la phrase veut dire c'est :

> Comment puis-je vérifier qu'une touche a été pressée de manière non bloquante ?

Tiens la [doc officielle de kbhit en francais](https://msdn.microsoft.com/fr-fr/library/58w7c94c.aspx) c'est vraiment de la merde, merci Microsoft :

> Le _kbhit fonction vérifie la console pour une séquence de touches récente.

Pourquoi pas celle d'au dessus alors :

> Comment puis-je vérifier qu'une touche a été pressée de manière non bloquante ?

ou

> Comment puis-je vérifier de manière non bloquante qu'une touche a été pressée  ?

þ

Reply:
Excellentes suggestions, merci. Je vais utiliser le 2ème